filebeat/docs/inputs/input-filestream-file-options.asciidoc

```diff
@@ -37,6 +37,24 @@ a `gz` extension:
 
 See <<regexp-support>> for a list of supported regexp patterns.
 
+===== `prospector.scanner.include_files`
+
+A list of regular expressions to match the files that you want {beatname_uc} to
+include. By default no files are excluded. This option is the counterpart of
+`prospector.scanner.exclude_files`.
+
+The following example configures {beatname_uc} to include files under `/var/log`:
```

Comment:
Its kind of an allow list. Maybe we can make this a little more clear? E.g. by on the example we could say: "The following example configures {beatname_uc} to ignore any files that are not store in /var/log`... or "allows users to collect files from /var/log only"...